### PR TITLE
Extract state card status to an element

### PR DESCRIPTION
**Pull request by andrey-git**
Extract state card status to an element.
This will allow reuse.

### CODE REVIEW

File: src/components/entity/state-status.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<link rel="import" href="../../../bower_components/polymer/polymer.html">
+
+<dom-module id="state-status">
+  <template>
+    <style>
```
Comment (balloob) [inline src/components/entity/state-status.html]: This component should not include any CSS. It should just render the text. Let the styling be done in whatever incorporates this element.

File: src/components/entity/state-status.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<link rel="import" href="../../../bower_components/polymer/polymer.html">
+
+<dom-module id="state-status">
```
Comment (balloob) [inline src/components/entity/state-status.html]: We should prefix our components with `hass-` to avoid collision

Comment (andrey-git) [inline src/components/entity/state-status.html]: renamed to ha-state-status, like most other components

**Comment by andrey-git** [comment]
Regarding CSS:
A webcomponent should be a building brick that you can optionally custom-style.
The included CSS is the way the status looks in HA by default. If needed, we can introduce variables in the future to override margin/text-height etc.

**Comment by balloob** [comment]
The smaller the components, the more reusable they are. Line height, margin and text alignment are all very specific to the place where the original component appeared.

I agree that font-size and color are 👍 

We can also remove the `<div>` in the template and style on `:host`

**Comment by andrey-git** [comment]
Should I create another component then that will contain the css and use `ha-state-status`?

Let's say someone wants to fork `state-card-display` while keeping the status appearance ;)
This forces the CSS needlesly to be forked too.

**Comment by balloob** [comment]
The browser incurs a cost for each web component. On mobile, this will be especially impactful. This is going to be a web component that is used frequently and thus breaking it out means another web component on the page for each entity that is available.

So I rather not break out a web component for no good reason (custom UI is not a good reason). Since a component that contains CSS like `margin` etc is not re-usable, there is no good reason.

Just copy paste the style for your custom UI.

**Pull request closed without merging**